pebbles/views/instances.py

```diff
@@ -202,6 +206,15 @@ def get(self, instance_id):
         instance.maximum_lifetime = blueprint.maximum_lifetime
         instance.cost_multiplier = blueprint.cost_multiplier
 
+        if arg_value.get('check_provisioning'):
```

Comment:
This whole block of code needs to be in the patch method, as discussed above. GET should only be used to return entries.

Reply:
We are not updating any resources so we cannot "PATCH". And we need  do provide a function(email) based on one particular value of the resource and not update anything so I had to settle with "GET".